themes/classic/_dev/js/components/top-menu.js

```diff
@@ -41,7 +41,7 @@ export default class TopMenu extends DropDown {
       }
       if (elmId && $(e.target).data('depth') === 0) {
         $(`#${elmId} .js-sub-menu`).css({
-          top: $(`#${elmId}`).height() + $(`#${elmId}`).position().top
+          top: $(`.js-sub-menu`).parent(`#${elmId}`).height() + $(`#${elmId}`).position().top
```

Comment:
Since it's an id and must be unique, why adding a `.js-sub-menu` with `parent` change? :thinking:

Reply:
The previous code with "$(`#${elmId}`)" only  wasn't working

Reply:
I think the problem from somewhere else.
Selecting a parent with this id is the same as `$(#${elmId})`

Reply:
I know. Calling only "$('#{elmId}')" set element to height:0 using the other code sets this one to height:29px like its siblings.

Reply:
So, we need to fix the problem where it start, not adding a weird patch :/